packages/scripts/src/utils/external-features.ts

```diff
@@ -33,6 +33,29 @@ export function getExternalFeaturesMetadata(
         return {
             name: defaultFeatureName!,
             envEntries,
+            externalRequests: extractDependencyMapping(defaultFeatureName!, features),
         };
     });
 }
+
+export function extractDependencyMapping(
```

Comment:
why this is exported?

Reply:
Because I am reusing this method in other projects